### DISCUSSION TITLE
当作terminal使用时启用本地应用 报错  “command not found”

### DISCUSSION THREAD

**Discussion by eastyy** [electerm/electerm#2658] [Q&A] [answered]
mac M1 客户端。
当我把electerm当作 terminal使用的场景下，运行脚本启动 Jekyll  报这样的错误。
我在mac 的terminal或者iterm2  客户端都可以正常运行。
请问我应该咋办？

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/12904350/194705283-9fba7dc7-7e17-4dcd-ba73-16d4c91c5b75.png">

**Comment by zxdong262** [comment, marked as answer]
设置里mac的exec设为zsh试试？

**Reply by eastyy** [reply]
感谢，按照你说的方法解决了。
第一次理解错了，先去看了mac的‘设置’。。。
<img width="1137" alt="image" src="https://user-images.githubusercontent.com/12904350/194720364-125d98e2-b789-4191-8424-5a5f57fed1a9.png">

**Comment by eastyy** [comment]
看了一下mac的shell官方默认是zsh。建议mac版本此处的默认值也改成zsh

**Reply by zxdong262** [reply]
新版本默认已经改为zsh
